cmd/snap/notes.go

```diff
@@ -42,12 +43,16 @@ func (n *Notes) String() string {
 		ns = append(ns, n.Price)
 	}
 
-	if n.Confinement != "" {
-		if n.Confinement != client.StrictConfinement {
-			ns = append(ns, n.Confinement)
+	devmodeSnap := n.Confinement != "" && n.Confinement != client.StrictConfinement
+	if n.Local {
+		if n.DevMode {
+			ns = append(ns, "devmode")
+		} else if devmodeSnap {
+			// snap is devmode, but is not installed in devmode
```

Comment:
Same. Your idea (which sounds good) was to have JailMode as a mode on itself, so we need to respect the idea when displaying too, right?